packages/at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart

```diff
@@ -267,7 +267,7 @@ class _AtOnboardingActivateScreenState
       } else if (e == AtOnboardingResponseStatus.serverNotReached) {
         await _showAlertDialog(
           e,
-          title: AtOnboardingLocalizations.current.error_server_not_found,
+          title: AtOnboardingLocalizations.current.msg_atSign_unreachable,
```

Comment:
Nice - thank you for updating the other alert dialogs also, not just the one mentioned in the issue